R/rdeephaven/R/update_by_ops_wrapper.R

```diff
@@ -427,24 +1123,48 @@ uby_rolling_avg_tick <- function(cols, rev_ticks, fwd_ticks = 0) {
   return(UpdateByOp$new(INTERNAL_rolling_avg_tick(cols, rev_ticks, fwd_ticks)))
 }
 
+#' @name
+#' uby_rolling_avg_time
+#' @title
+#' Rolling average with time as the windowing unit
+#' @md
+#'
 #' @description
 #' Creates a simple moving average (SMA) UpdateByOp for each column in `cols`, using time as the windowing unit.
-#' This uses ISO-8601 time strings as the reverse and forward window parameters.
+#'
+#' @details
+#' This uses [ISO-8601](https://en.wikipedia.org/wiki/ISO_8601) time strings as the reverse and forward window parameters.
 #' Negative values are allowed and can be used to generate completely forward or completely reverse windows.
 #' A row containing a null value in the timestamp column belongs to no window and will not be considered
 #' in the windows of other rows; its output will be null.
 #' Here are some examples of window values:
-#'   `rev_time = "PT00:10:00", fwd_time = "PT00:10:00"` - contains rows from 10m before through 10m following
+#' - `rev_time = "PT00:10:00", fwd_time = "PT00:10:00"` - contains rows from 10m before through 10m following
 #'     the current row timestamp (inclusive)
-#'   `rev_time = "PT00:10:00", fwd_time = "-PT00:05:00"` - contains rows from 10m before through 5m before the
+#' - `rev_time = "PT00:10:00", fwd_time = "-PT00:05:00"` - contains rows from 10m before through 5m before the
 #'     current row timestamp (inclusive), this is a purely backwards looking window
-#'   `rev_time = "-PT00:05:00", fwd_time = "PT00:10:00"` - contains rows from 5m following through 10m
+#' - `rev_time = "-PT00:05:00", fwd_time = "PT00:10:00"` - contains rows from 5m following through 10m
 #'     following the current row timestamp (inclusive), this is a purely forwards looking window
+#'
+#' This function acts on aggregation groups specified with the `by` parameter of the `update_by()` caller function.
+#' The aggregation groups are defined by the unique combinations of values in the `by` columns. For example,
+#' if `by = c("A", "B")`, then the aggregation groups are defined by the unique combinations of values in the
+#' `A` and `B` columns.
+#'
+#' This function, like other Deephaven `uby` functions, is a generator function. That is, its output is another
+#' function called an [`UpdateByOp`][UpdateByOp] intended to be used in a call to `update_by()`. This detail is typically
+#' hidden from the user. However, it is important to understand this detail for debugging purposes, as the output of
+#' a `uby` function can otherwise seem unexpected.
+#'
 #' @param ts_col String denoting the column to use as the timestamp.
 #' @param cols String or list of strings denoting the column(s) to operate on. Can be renaming expressions, i.e. “new_col = col”.
-#' Default is to compute the rolling average for all non-grouping columns.
+#' Default is to compute the rolling sum for all non-grouping columns.
```

Comment:
you've changed group to sum, and average to sum here - just checking that's deliberate